app/views/layouts/_facebook_script.html.erb

```diff
@@ -1,21 +1,75 @@
 <!-- Facebook script -->
-<div id="fb-root"></div>
 <script>
-window.fbAsyncInit = function() {
-  FB.init({
-    appId      : '<%= ENV["FACEBOOK_APP_ID"] %>',
-    xfbml      : true,
-    version    : 'v2.8'
+var bindFacebookEvents, defer, dispatch, handleEvent, initializeFacebookSDK;
+var loadFacebookSDK, loaded, restoreFacebookRoot, saveFacebookRoot;
+
+({defer, dispatch} = Turbolinks);
+
+loaded = false;
```

Comment:
I don't trust global variables. Especially with a name that's so generic: any script we load might use the same variable.

I recommend you add a variable to `window` with a very specific name, like `window.pangeasContentLoaded`.

In the future, if we end up having a lot of JS code, we can create a `window.Pangeas` object, containing all variables we need. But I don't think we'll come to that: we'll start using React before.